src/Elcodi/Component/Media/Entity/Traits/ImagesContainerTrait.php

```diff
@@ -67,6 +74,39 @@ public function getImages()
     }
 
     /**
+     * Get sorted images
+     *
+     * @return \Doctrine\Common\Collections\ArrayCollection
+     */
+    public function getSortedImages()
```

Comment:
What's the rationale behind this one? I found the code is not easy enough to follow.
Also, is this a trait for entities? I thought no logic could go inside entites, only getters and setters.